### PR TITLE
Typos and Definition addition

### PR DESCRIPTION
**Pull request by complexbrains**
### Summary

Fixes #<NUM>

Several typos are corrected and an amendment has been done in one of the definition in the glossary. 

### List of changes proposed in this PR (pull-request)

Several typos are corrected and an amendment has been done in one of the definition in the glossary. 

* *Lorem ipsum dolor sit amet, consectetur adipiscing.*
* *Lorem ipsum dolor sit amet, consectetur adipiscing.*

### What should a reviewer concentrate their feedback on?
Whether the correction and amendment made suits with the aim of the glossary or not. 

- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
- [x] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Review by paulowoicho — APPROVED** [review]
Thanks @complexbrains! This looks great! Please merge this when you're ready

**Comment by paulowoicho** [comment]
@all-contributors, please add @complexbrains for fixing bugs

**Comment by paulowoicho** [comment]
@all-contributors please add @complexbrains for bug fixing

File: book/website/afterword/glossary.md
Comment (malvikasharan) [inline book/website/afterword/glossary.md:20]:
```
Authors in this context are the contributors to _The Turing Way_ project who have made a substantial contribution to the project such as writing a subchapter, facilitating community interactions, maintaining project’s infrastructure, mentoring contributions, and supporting others participation.
All authors are named as co-authors on the book as a whole.
```

Comment (malvikasharan) [reply book/website/afterword/glossary.md:20]: Author's definition here is specifically for The Turing Way project. Please update the definition as per our acknowledgement chapter.

Comment (complexbrains) [reply book/website/afterword/glossary.md:20]: I see, I was thinking it is missing some important information to cover the whole scope of authorship. The above definition you added reflects it better. I will make the change and will do the PR then. Thank you.

Comment (malvikasharan) [reply book/website/afterword/glossary.md]: Thank you so much. 💫

**Comment by malvikasharan** [comment]
@paulowoicho I am merging this as I think @complexbrains doesn't have write access to the repo and hence can't merge it. Thank you @complexbrains for the edits 👍

**Pull request merged**
